packages/core/src/auth/tokens/token-scope.ts

```diff
@@ -5,13 +5,13 @@ export enum CacheRole {
 }
 
 export class All {}
```

Comment:
now that I see that the class definition for `All` is empty, I think you can just do `export const` instead of `export class`, and then you won't need the `new All` stuff. You will have to choose a type for it that can be disambiguated from `String` though. could be an enum or an integer. or you can still use a class and just make the class private, and have `All` be a constant instance of the class.

Reply:
Done in https://github.com/momentohq/client-sdk-javascript/pull/598/commits/f2aad35500cbec9f6c42ca858e70abafffacbf58